frontend/src/components/form/ColorPicker.vue

```diff
@@ -0,0 +1,76 @@
+<!--
+Displays a field as a color picker (can be used with v-model)
+-->
+
+<template>
+  <div>
+    <v-menu
+      ref="menu"
+      v-model="showColorPicker"
+      :close-on-content-click="false"
+      transition="scale-transition"
+      offset-y
+      min-width="290px">
+      <template v-slot:activator="{on}">
+        <v-text-field
+          v-model="color"
+          v-bind="$attrs"
+          readonly
```

Comment:
```suggestion
          readonly
          hide-details="auto"
```
Ab Vuetify 2.2.0 kann das Feld ausgeblendet werden, wenn keine Message steht.

Reply:
Ok, dann mach ich das aber gleich bei sämtlichen Komponenten.

Reply:
Hey, die Komponten stehen dann aber schon sehr Nahe aneinander:
![image](https://user-images.githubusercontent.com/449555/75804322-12c02200-5d80-11ea-9003-b4ee0564716d.png)

Reply:
Ja aber ich mache lieber einen definierten Abstand auf dem Wrapper als überall den gleich riesigen Abstand durch das Fehlen der Message, wo man dann nicht mehr weis zu 
welchem Feld die Message dazugehört.

Reply:
Ja aber ich mache lieber einen definierten Abstand auf dem Wrapper als überall den gleich riesigen Abstand durch das Fehlen der Message, wo man dann nicht mehr weis zu 
welchem Feld die Message dazugehört.